test/specs/comment/tests/01.comment.js

```diff
@@ -0,0 +1,30 @@
+exports.execute = function (options) {
+
+    describe('For checking the .comment on models, ', function () {
+        var schemaName = "schema_with_comment", schema;
+
+        beforeAll(function (done) {
+            schema = options.catalog.schemas.get(schemaName);
+            done();
+        });
+
+        // Test Cases:
+        it('schema_with_comment should have a non-null .comment property.', function () {
+            expect(schema.comment).toBe("schema with a comment");
```

Comment:
@chiragsanghvi Ok I added checks for (1) `<model>.hasOwnProperty('comment')`; and (2) `null` values in `.comment` when the schema doesn't have a comment.